awesomerc.lua.in

```diff
@@ -261,6 +265,7 @@ globalkeys = awful.util.table.join(
     awful.key({ modkey, "Control" }, "l",     function () awful.tag.incncol(-1)         end),
     awful.key({ modkey,           }, "space", function () awful.layout.inc( 1) end),
     awful.key({ modkey, "Shift"   }, "space", function () awful.layout.inc(-1) end),
+    awful.key({ modkey,           }, "s", function () awful.layout.suit.corner.swap_privileged(); awful.layout.arrange(mouse.screen) end),
```

Comment:
I am not sure if this binding is useful/unspecific enough to have it by default.
If so, it should be also added to the man page etc.